docs/pages/reference/deployment/backends.mdx

```diff
@@ -126,14 +126,85 @@ To configure Teleport for using etcd as a storage backend:
 - Follow [etcd's cluster hardware recommendations](https://etcd.io/docs/v3.6/op-guide/hardware/). In particular, leverage
   SSD or high-performance virtualized block device storage for best performance.
 - Install etcd and configure peer and client TLS authentication using the [etcd
-  security guide](https://etcd.io/docs/v3.6/op-guide/security/).
-  - You can use [this script provided by
-    etcd](https://github.com/etcd-io/etcd/tree/master/hack/tls-setup) if you
-    don't already have a TLS setup.
+  security guide](https://etcd.io/docs/v3.6/op-guide/security/). See
+  [Configuring mTLS](#configuring-mtls-authentication-with-etcd) below for
+  details.
 - Configure all Teleport Auth Service instances to use etcd in the "storage" section of the config file as shown below.
 - Deploy several Auth Service instances connected to etcd backend.
 - Deploy several Proxy Service instances that have `auth_server` pointed to the Auth Service to connect to.
 
+### Configuring mTLS authentication with etcd
+
+We recommend using [cfssl](https://github.com/cloudflare/cfssl) and etcd's
+[`tls-setup` script](https://github.com/etcd-io/etcd/tree/main/hack/tls-setup)
+to configure mTLS between Teleport and etcd. It's helpful for creating
+certificates and keys for the entire etcd cluster. After [installing
+cfssl](https://github.com/cloudflare/cfssl?tab=readme-ov-file#installation),
+clone the `tls-setup` repo and follow the `Readme.md` file to customize the
+script configuration. Make sure to tweak the makefile if you change the number
+of etcd nodes (and thus, the number of `infra` variables).
+
+After you run `make`, the following files will be generated:
+
+- `tls-setup/certs/ca.pem` is the certificate authority and it should be
+  referenced in Teleport's `tls_ca_file` option (see [Configuring Auth Service
+  instances to use etcd](#configuring-auth-service-instances-to-use-etcd)
+  below).
+- `tls-setup/certs/peer-*` files are used internally by etcd for communicating
+  between nodes.
+- `tls-setup/certs/<host>.pem` and `tls-setup/certs/<host>-key.pem` are the
+  host certificates and their corresponding private keys. Each corresponding
+  etcd host needs to know them, and they are required for providing transport
+  security on the etcd side.
+
+This should be enough to configure mTLS on etcd side, but to configure it on
+Teleport side, you need one more thing: client certificates. Here's one way to
+do it:
+
+1. Generate a default CSR:
+   ```sh
+   cfssl print-defaults csr > client.json`
+   ```
```

Comment:
```suggestion
   ```code
   $ cfssl print-defaults csr > client.json`
   ```
```
For code fences with commands, we use the `code` label to render our custom code fence component. The component expects commands to include a `$` prompt.